### ISSUE TITLE
False positive multiline values

### ISSUE THREAD

**Issue by orgads** [antw/iniparse#14]
When a line ends with an *escaped* backslash, it should not be considered a multiline value.

```ini
[Section]
Test1=foo\\
Test2=bar
```

```ruby
ini = IniParse.parse(File.read('file.ini'))
puts ini.to_ini
```

```ini
[Section]
Test1 = foo\Test2=bar
```

**Comment by antw** [comment]
Thanks for the report. Should be fixed in v1.4.1 which I just pushed to rubygems.org.

**Comment by orgads** [comment]
That was fast! Thanks!

It is not accurate though, since if you have 3 (or any other odd number of) backslashes it should be joined, but that's a rare use-case.
